assessment.md

```diff
@@ -119,56 +119,57 @@ when creating python programs.
   * E = Use of multiple useful branches/forks demonstrated appropriately
-    containing multiple coherent and descriptive commits
+    containing multiple consistent and descriptive commits
 * Creating and using a repository branch
   * N = Did not create or utilize branches
-  * I = Create unclear branches
-  * A = Create too many poorly named branches
-  * G = Create useful adequately named branches
-  * E = Create useful and descriptive branches
+  * I = Created unclear branches
+  * A = Created too many poorly named branches
+  * G = Created useful adequately named branches
+  * E = Created useful and descriptive branches
 * Creating and using a repository fork
   * N = Did not create or utilize forks
-  * I = Create unclear forks
-  * A = Create too many poorly named forks
-  * G = Create useful adequately named forks
-  * E = Create useful and descriptive forks
+  * I = Created unclear forks
+  * A = Created too many poorly named forks
+  * G = Created useful adequately named forks
+  * E = Created useful and descriptive forks
 * Merging a branch or a fork to another branch or fork
   * N = Not communicating changes before merging branches and forks
-  * I = Merging with minimal communication or merging faulty code
+  * I = Merging with minimal communication or merging defective code
   * A = Communicating changes with the team and reconciling differences
   * G = Merging branches after extensive communication
   * E = Merging branches with well-documented, detailed code and
-    extensive communication
+    extensive team communication
 * Creating and reviewing a pull request
-  * N = No PR made
+  * N = No PR(Pull Request) made
   * I = Does not tag the related issues it is trying to close with poor description
   * A = Tags any related issues it is trying to close, describes
     proposed changes by a person
   * G = Tags any related issues it is trying to close, describes most proposed changes,
-    discusses results of PR with Travis CI
+    and passes Travis CI
   * E = Tags any related issues it is trying to close, fully describes all
     proposed changes and passes Travis CI
 * Using appropriate commit messages
-  * N = Commit messages are nonsensical
+  * N = Commit messages do not make sense
   * I = Commit messages are uninformative
   * A = Commit messages are sensible but not descriptive
-  * G = Commit messages are both clear, and moderately descriptive
-  * E = Commit messages are clear, concise and descriptive
+  * G = Commit messages are clear, concise and moderately descriptive
+  * E = Commit messages are clear, concise, and descriptive
 
 ### Continuous Integration with Travis
 
 * Setup and configure Travis CI
   * N = Fails to perform any setup and configuration of Travis CI
   * I = Sets up and configures Travis CI which performs no checks
-  * A = Sets up and configures Travis CI without performing checks on less
+  * A = Sets up and configures Travis CI that performs checks on less
```

Comment:
Wait.... this inverts the meaning of the sentence, and seems that setting up checks on less relevant project deliverables is not as desirable? This is not clear.